src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -317,6 +317,15 @@ module sonic-device_metadata {
                     description "Enable syslog with OS version feature.";
                     default "false";
                 }
+
+                leaf zebra_nexthop {
```

Comment:
Why can't we reuse nexthop_group that is already available in device metadata?

Reply:
Thanks, I am not sure what the nexthop_group flag is for, does it indicate the whole feature status? Can we also use it for the zebra disable nexthop case？ thanks a lot

Reply:
@eddieruan-alibaba  can clarify this but i believe it is to indicate the whole feature https://github.com/sonic-net/SONiC/pull/1425